planner_cspace/src/distance_map.cpp

```diff
@@ -138,7 +136,7 @@ void DistanceMap::fillCostmap(
             }
             if (collision)
             {
-              edges_local[p] = true;
+              edges_buf_[p[0] + p[1] * p_.size[0]] = true;
```

Comment:
https://en.cppreference.com/w/cpp/container/vector_bool
> Does not guarantee that different elements in the same container can be modified concurrently by different threads. 

This line is executed by multiple threads

Reply:
I added temporal storage named `std::vector<size_t> edge_indexes`.

Reply:
I think the performance degradation is acceptable.
```
[1690797074.360340821, 1690279749.841883353][/planner_3d]: New goal received (266, 513, 8)
1690797074.424991012, 1690279749.902270411][/planner_3d]: Cost estimation cache generated (0.0646 sec.)
[1690797074.431511267, 1690279749.912324169][/planner_3d]: Map updated
[1690797074.433128297, 1690279749.912324169][/planner_3d]: Costmaps updated (0.0016)
[1690797074.455404871, 1690279749.932516006][/planner_3d]: Cost estimation cache search queue initial size: 46695, capacity: 60704
[1690797074.455429775, 1690279749.932516006][/planner_3d]: Cost estimation cache updated (0.0222 sec.)
[1690797074.471341985, 1690279749.952620123][/planner_3d]: Trying to publish message of type [geometry_msgs/PoseStamped/d3812c3cbc69362b77dc0b19b345f8f5] on a publisher with type [geometry_msgs/PoseStamped/d3812c3cbc69362b77dc0b19b345f8f5]
[1690797074.501594817, 1690279749.972727846][/planner_3d]: Path found (0.0302 sec.)     
[1690797074.501627082, 1690279749.972727846][/planner_3d]: Trying to publish message of type [geometry_msgs/PoseArray/916c28c5764443f268b296bb671b9d97] on a publisher with type [geometry_msgs/PoseArray/916c28c5764443f268b296bb671b9d97]
[1690797074.508748404, 1690279749.972727846][/planner_3d]: Hysteresis map generated (0.0071 sec.)
[1690797074.556863731, 1690279750.037711948][/planner_3d]: Map updated
[1690797074.559274794, 1690279750.037711948][/planner_3d]: Costmaps updated (0.0023)
[1690797074.580336898, 1690279750.057821310][/planner_3d]: Cost estimation cache search queue initial size: 46685, capacity: 60704
[1690797074.580366817, 1690279750.057821310][/planner_3d]: Cost estimation cache updated (0.0210 sec.)
[1690797074.756607771, 1690279750.238098679][/planner_3d]: Map updated
[1690797074.758966322, 1690279750.238098679][/planner_3d]: Costmaps updated (0.0023)    
[1690797074.781845387, 1690279750.258207917][/planner_3d]: Cost estimation cache search queue initial size: 46685, capacity: 60704
[1690797074.781878368, 1690279750.258207917][/planner_3d]: Cost estimation cache updated (0.0228 sec.)
[1690797074.999613759, 1690279750.475198153][/planner_3d]: Path found (0.0260 sec.)     
[1690797075.008093280, 1690279750.485255865][/planner_3d]: Hysteresis map generated (0.0084 sec.)
[1690797075.008279763, 1690279750.485255865][/planner_3d]: Map updated
[1690797075.009892748, 1690279750.485255865][/planner_3d]: Costmaps updated (0.0016)    
[1690797075.031660764, 1690279750.505362525][/planner_3d]: Cost estimation cache search queue initial size: 46685, capacity: 60704
[1690797075.031693066, 1690279750.505362525][/planner_3d]: Cost estimation cache updated (0.0217 sec.)

```